src/clientversion.cpp

```diff
@@ -71,7 +71,7 @@ std::string FormatSubVersion(const std::string& name, int nClientVersion, const
 
 std::string CopyrightHolders(const std::string& strPrefix)
 {
-    const auto copyright_devs = strprintf(_(COPYRIGHT_HOLDERS).translated, COPYRIGHT_HOLDERS_SUBSTITUTION);
+    const auto copyright_devs = strprintf(_(COPYRIGHT_HOLDERS), COPYRIGHT_HOLDERS_SUBSTITUTION).translated;
```

Comment:
re: https://github.com/bitcoin/bitcoin/pull/31072#discussion_r1845404225

Right, I think it would be bad if changing or tweaking the name of the program required newly translating "The %s developers" with the new name of the program hardcoded.